.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -9,19 +9,38 @@ body:
   - type: checkboxes
     attributes:
       options:
-      - label: I searched for similar bug reports and found none was relevant.
+      - label: I searched for similar bug reports in https://github.com/Botspot/pi-apps/issues and found none was relevant.
         required: true
+      - label: I tried updating pi-apps and the issue still not fixed.
+        required: false
   - type: input
     id: desc
     attributes:
       label: What happened?
-      description: Explain what what happened.
-      placeholder: "Ex. 'App' won't install"
+      description: Briefly explain what happened.
+      placeholder: "Ex. 'App' won't install, 'App' unable to launch after update"
```

Comment:
```suggestion
      placeholder: Ex. 'App' won't install, 'App' unable to launch after update
```

Reply:
Same as above, there are two `'` in the sentence, and YAML can't really recognize them.

Reply:
So leave the quotes. I guess we don't really neednto remove them.